Dockerfile

```diff
@@ -99,7 +93,7 @@ RUN yum install --skip-broken -y perl \
         libjpeg-devel libpng-devel \
 # backup ubi release info
         && mkdir /tmp/release && mv /etc/*release* /tmp/release \
-        && rpm -Uvh --force /tmp/centos-release-7-7.1908.0.el7.centos.x86_64.rpm && sed -i 's/$releasever/7/g' /etc/yum.repos.d/* \
+        && rpm -Uvh --force /tmp/centos-release-7-7.1908.0.el7.centos.${ARCH}.rpm && sed -i 's/$releasever/7/g' /etc/yum.repos.d/* \
```

Comment:
can this work on power?

Reply:
@haoqing0110  If we add centos-release-7-7.1908.0.el7.centos.ppc64le.rpm in external-deps folder here https://github.com/rahularora19911/management-ingress/tree/s390x/external-deps 
It will work on power as well. Please let me know if you want me to include that also in my PR. I can do that. I can see the rpm present here. https://mirror.chpc.utah.edu/pub/vault.centos.org/altarch/7.7.1908/os/ppc64le/Packages/centos-release-7-7.1908.0.el7.centos.ppc64le.rpm
Thanks.

Reply:
@rahularora19911 thanks and plz help add ppc64le.rpm, so that won't block ppc64le downstream build. 
/cc @arewm @dberg1

Reply:
@haoqing0110  I have added the same. Thanks

Reply:
FYI, downstream does not use `dumb-init` at all so this rpm should not installed by the downstream Dockerfile.

Reply:
I have discussed this point offline with @dberg1 , we shall not install `dumb-init` in downstream [Dockerfile](https://pkgs.devel.redhat.com/cgit/containers/management-ingress/tree/Dockerfile?h=rhacm-2.3-rhel-8) Thanks.